core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -662,7 +662,6 @@ protected abstract function tearDown();
   public function errorHandler($severity, $message, $file = NULL, $line = NULL) {
     if ($severity & error_reporting()) {
       $error_map = array(
-        E_STRICT => 'Run-time notice',
```

Comment:
Do we also need E_STRICT for PHP version < 8.4 for BackdropTestCase::errorHandler()?

Reply:
Thanks, I restored E_STRICT the same as we did in `backdrop_error_levels()`.